portal-web/test/functional/com/liferay/portalweb/tests/MyTestFile.testcase

```diff
@@ -0,0 +1,48 @@
+@component-name = "portal-web-forms-and-data-lists"
+definition {
+    property testray.main.component.name = "Forms";
+
+    setUp {
+        TestCase.setUpPortalInstance();
+
+        User.firstLoginPG();
+
+        Navigator.openURL();
+    }
+
+    tearDown {
+        User.logoutAndLoginPG(
+            userLoginEmailAddress = "test@liferay.com",
+            userLoginFullName = "Test Test");
+    }
+
+    @description = "LRQA-82607. Access Portal Global Menu and validate CONFIGURATION string at Control Panel"
+    @priority = "5"
+    test ValidateConfigurationStringAtControlPanel {
+		ApplicationsMenuHelper.openApplicationsMenu();
+
+		ApplicationsMenu.gotoPanel(panel = "Control Panel");
+
+        AssertElementVisible(locator1 = "ApplicationsMenu#CONFIGURATION_SECTION_TITLE");
+    }
+
+    @description = "LRQA-82607. Set Screen size to custom and validate values"
+    @priority = "5"
+    test SetScreenSizeAndValidateValues {
+		MobileDevice.previewPGWithCorrectCustomNumbers(
+            height1 = 1,
+            height2 = 920,
+            width1 = 1,
+            width2 = 080);
+
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
+		Click.waitForMenuToggleJSClick(locator1 = "ControlMenu#SIMULATION");
```

Comment:
Can you check if clicking twice is actually necessary? If not, we can adapt the steps or just remove this one.

Reply:
Clicking twice is necessary. The first click closes the menu and the second one reopens it.

Reply:
So maybe a solution is: create a if conditional to check if some element of the sidebar is not displayed. If true, then click the button. For example:
```
if (IsElementNotPresent(key_selectFieldLabel = "Product List Renderer", locator1 = "Select#ANY")) {
	Click(
		key_text = "Product List Renderer",
		locator1 = "Link#ANY");
}
```

Reply:
The problem is the test is supposed to close and reopen it. That's what we're testing: If when I close the sidebar and reopen it, the values stay the same. A conditional like that would open the sidebar if it's closed (which it won't be) and not do anything if it's already open. That's not what I need. I need the sidebar to close and open again.